### ISSUE TITLE
[Feature suggestion] backup Windows Subsystem for Android apps and data

### ISSUE THREAD

**Issue by AlecMcCutcheon** [alesimula/wsa_pacman#42]
WSATools is going to be getting this Feature soon but I like your tool more so if you added this we wouldn't have to have 2 apps for managing our WSA install.

**Comment by AlecMcCutcheon** [comment]
Also you should add a button to open the android settings

**Comment by alesimula** [comment]
@AlecMcCutcheon there's already a button to open the settings

**Comment by AlecMcCutcheon** [comment]
I don't mean the Windows 10 wsa settings I mean like the Android settings

**Comment by AlecMcCutcheon** [comment]
> @AlecMcCutcheon there's already a button to open the settings

Oh i see that now by bad

**Comment by akuropka** [comment]
@AlecMcCutcheon, you can backup and restore `%LocalAppData%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\userdata.vhdx` manually.

I was using [WSATools](https://github.com/Simizfo/WSATools) in the beginning... it was the first one of its kind and is work of Simone both make it famous. Aside of that it is closed source and wsa_pacman is overall more smooth in my idea.

What I am missing here is... [GitHub Sponsors](https://github.com/sponsors)

**Comment by AlecMcCutcheon** [comment]
You're right. It Doesn't have to have all kinds of features,  And thank you for the tip I had no idea there was just a virtual disk for that.  Any idea what extensions are used in VS code in order to get the project booted up myself I would be happy to contribute code myself.  I have tried to load it up yet but it's just asking before I dig into it

**Comment by alesimula** [comment]
@AlecMcCutcheon I just use the [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code), the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) and the [vscode-proto3 library](https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3) (only used for the \proto\options.proto file).

Of course you must install Flutter and Visual Studio SDK

The project will initially report some errors (missing files), you must run it anyway and it will auto-generate the localization classes
